Tools/WebKitTestRunner/cocoa/TestControllerCocoa.mm

```diff
@@ -676,6 +677,15 @@ static WKContentMode contentMode(const TestOptions& options)
         [webpagePreferences _setNetworkConnectionIntegrityPolicy:_WKWebsiteNetworkConnectionIntegrityPolicyNone];
 #if PLATFORM(IOS_FAMILY)
     [webpagePreferences setPreferredContentMode:contentMode(options)];
+
+    if (options.keyboardDismissalGestureEnabled()) {
+        for (_WKFeature *feature in WKPreferences._features) {
+            if ([feature.key isEqualToString:@"KeyboardDismissalGestureEnabled"]) {
+                [[configuration preferences] _setEnabled:YES forFeature:feature];
+                break;
+            }
+        }
+    }
```

Comment:
Why is this necessary? I don't see this done for other flags.

I think you might need to refer to the flag as `KeyboardDismissalGestureEnabled` (uppercase), rather than `keyboardDismissalGestureEnabled`, in `TestOptions.cpp` and the test file.

Reply:
Oh, good catch — using an uppercase `K` there ought to Just Work™ without this extra logic.

Thanks for the review!